packages/donations-app/app/components/home/homeController.js

```diff
@@ -1,19 +1,43 @@
 angular.module('aliceApp')
-  .controller('HomeController', ['ProjectService', 'NotificationService', 'API', '$location', '$http', function (ProjectService, NotificationService, API, $location, $http) {
+  .controller('HomeController', ['ProjectService', 'NotificationService', 'API', '$location', '$http', '$uibModal', 'MODE', function (ProjectService, NotificationService, API, $location, $http, $uibModal, MODE) {
     var vm = this, navHeight = $('#navbar-menu').height();
 
     var loadData = function () {
       vm.contact = {};
       loadProjectsWithCharities();
     };
 
-    var loadProjectsWithCharities = function () {
-      ProjectService.getActiveProjects().then(function (projects) {
-        vm.projects = _.map(projects.data, (project) => {
-          let keyList = ['title', 'code', 'charity', 'lead', 'img'];
+    if (MODE != 'prod') {
+      $uibModal.open({
+        templateUrl: '/components/vodafone/homePageModal.html',
+        backdrop: 'static',
+        resolve: {}
+      });
+    }
 
-          return _.pick(project, keyList);
-        });
+    // FIXME
+    // Livia asked to move fusion-housing project from the first position of
+    // active projects
+    // The problem is connected with the fact that fusion housing project was
+    // the second project created on stage (after st-mungos)
+    function doHackForFusionHousing(projects) {
```

Comment:
It'll be better to add an order field to the project, similar to the improvement planned for outcomes.
It's not urgent, but if we want to postpone it, let's add a jira issue for it.

Reply:
I have created JIRA task https://alice-si.atlassian.net/browse/AD-394